BookOfHabits/BookOfHabitsMicroservice.Application.Service/InstallCardApplicationService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BookOfHabitsMicroservice.Application.Models.Habit;
+using BookOfHabitsMicroservice.Application.Services.Abstractions;
+using BookOfHabitsMicroservice.Application.Services.Abstractions.Exceptions;
+using BookOfHabitsMicroservice.Application.Services.Implementations.Base;
+using BookOfHabitsMicroservice.Domain.Entity;
+using BookOfHabitsMicroservice.Domain.Repository.Abstractions;
+
+namespace BookOfHabitsMicroservice.Application.Services.Implementations
+{
+    public class InstallCardApplicationService(IRepository<Person, Guid> personRepository,
+                                                IRepository<Habit, Guid> habitRepository,
+                                                IRepository<Card, Guid> cardRepository) : BaseService, IInstallCardApplicationService
+    {
+        public async Task InstallCardAsync(InstallCardModel installCardModel, CancellationToken token = default)
+        {
+            Person owner = await personRepository.GetByIdAsync(x => x.Id.Equals(installCardModel.PersonId), cancellationToken: token)
+                ?? throw new NotFoundException(FormatFullNotFoundErrorMessage(installCardModel.PersonId, nameof(Person)));
+
+            Habit habit = await habitRepository.GetByIdAsync(x => x.Id.Equals(installCardModel.HabitId), includes: $"{nameof(Habit.Owner)};{nameof(Habit.Card)}", cancellationToken: token)
```

Comment:
did you check this one and it works properly? In implementation you have mentioned separator like ',', but here you have made separator between entities like ';'

Reply:
Thanks for the tip, there's a mistake.